package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "orca",
-  "version": "1.0.0",
+  "version": "1.0.1-rc.1",
```

Comment:
Did you base your branch off `master` or #103 ? If the former, we should stick to `1.0.0` here.